aws/resource_aws_acm_certificate_validation_test.go

```diff
@@ -31,64 +25,225 @@ func TestAccAwsAcmCertificateValidation_basic(t *testing.T) {
 		Steps: []resource.TestStep{
 			// Test that validation times out if certificate can't be validated
 			resource.TestStep{
-				Config:      testAccAcmCertificateValidation_basic(domain),
+				Config:      testAccAcmCertificateValidation_timeout(domain),
```

Comment:
Nitpick: I'd expect `_basic` test to be _actually_ basic and this timeout logic to be exercised perhaps in a separate test.

Reply:
Valid point, I'll separate